source/localizable/tutorial/autocomplete-component.md

```diff
@@ -144,11 +140,11 @@ We also define a `search` action here that is passed in to the component,
 For these actions to work, we need to modify the Mirage `config.js` file 
 to look like this, so that it can respond to our queries.
 
-```app/mirage/config.js
+```mirage/config.js
 export default function() {
   this.get('/rentals', function(db, request) {
     let rentals = [{
-        type: 'rentals',
```

Comment:
I know this works from the super rentals repo. I'll try plural and look in the spec to see